skorch/tests/test_classifier.py

```diff
@@ -166,7 +166,7 @@ def test_not_fitted_raises(self, net_cls, module_cls, data, method):
                "before using this method.")
         assert exc.value.args[0] == msg
 
-    @flaky(max_runs=3)
+    @flaky(max_runs=5)
```

Comment:
That is getting pretty flaky indeed.

Reply:
I made a grid search (yay) to find better parameters. Hopefully the test is less flaky now.